### ISSUE TITLE
'BorderlessGaming.Logic.dll' could not be found on Build

### ISSUE THREAD

**Issue by Marco-MP** [Codeusa/Borderless-Gaming#433]
Hi there,

_so first of all I would like to apologize if this is the wrong place for this kind of issue but since the in the Readme referenced Website for Help (andrew.im) seems to be down from here, I saw no other place to put this.
If you know where I should ask this instead, please be so kind and redirect me._

On to the Topic:

I am failing so far at building a working version. Every time I try to build the Code, it seems, that I'm missing the BorderlessGaming.Logic.dll and an thus a lot of components.

I'm kinda certain that somewhere during the VS2017 Install I must've failed to choose the right Workload or missed something else. However I looked through there multiple times and wasn't able to find anything I might have missed, so I would like to ask you guys here for an answer.

**I did the following:**
- I installed Visual Studio 2017 Community Edition (Version 15.9.7)
- During install, I chose to install the ".Net Desktop Development" Workload because it listed C# as subpoint and due to the C# code here I was convinced I would need atleast a C# Compiler which I found there.
- From here on I followed the Readme (clone Repo, open Solution File)
- I chose Release in this Dropdown Menu which I guessed might be the right one?:
![devenv_2019-02-24_12-56-18](https://user-images.githubusercontent.com/27819643/53298845-ae48b980-3833-11e9-8968-565ef33a0e76.png)
- I build the Solution with the only Option I found for that.
![devenv_2019-02-24_12-57-33](https://user-images.githubusercontent.com/27819643/53298857-c91b2e00-3833-11e9-86ce-ba4097df158d.png)

Those steps resulted in the following Errors and Warnings:
![unbenannt](https://user-images.githubusercontent.com/27819643/53298693-db946800-3831-11e9-901b-031d30b28912.PNG)

I would appreciate any form of help I might get from you and thank you already in advance :).

**Comment by sfuantos** [comment]
You need to rename the variable from "task" to "td", there is already a pull request (Fixed compile error, changing "task" to "td" #431) to fix this issue.

**Comment by Marco-MP** [comment]
Thank you for your help.
I got it working now!

However #431 was merely the start of the existing problems I had to resolve to get it working.
after I checked out the PR I faced three further issues which were ultimately only to be resolved by removing the "post-build-Calls" of signtool and iscc.

since I now had an .exe that I could launch, I did so and ran into the next issue which was: #349 

After fixing this the way it is suggested there in the comments, I got a working version.

Since this specific issue is resolved through PR #431, I will close it.
